src/families/filecoin/bridge/utils/addresses.ts

```diff
@@ -0,0 +1,150 @@
+import base32Decode from "base32-decode";
```

Comment:
some CI fail here.

Reply:
I added the missing dependency on package.json